packages/user-service/src/cron.ts

```diff
@@ -0,0 +1,46 @@
+import * as _ from 'lodash';
+import * as async from 'async';
+import { User } from './schema';
+import moment from 'moment';
+import { UserGroupAPIHelper } from './helpers';
+/**
+ * @class UserSyncCron
+ */
+export class UserSyncCron {
+  public syncUsers() {
+    User.find().then((userInfo: any[]) => {
+      if (userInfo.length) {
+        let rhatUUIDs: string[] = [];
+        rhatUUIDs = userInfo.map((user: UserType) => user.rhatUUID);
+        if (rhatUUIDs.length) {
+          async.map(rhatUUIDs, async rhatUUID => {
```

Comment:
No need for the callback to be async. It is needed only when you need to use await.

```suggestion
          async.map(rhatUUIDs, rhatUUID => {
```

Reply:
Changed it